mpf/devices/ball_device/pulse_coil_ejector.py

```diff
@@ -73,11 +73,13 @@ def ball_search(self, phase, iteration):
                 return self._fire_coil_for_search(True)
         elif phase == 2:
             # round 2: all devices except trough. small pulse
-            if 'trough' not in self.ball_device.config['tags']:
+            if 'trough' not in self.ball_device.config['tags'] and \
+                    'no-eject' not in self.ball_device.config['tags']:
                 return self._fire_coil_for_search(False)
```

Comment:
Looks good! Maybe we can still pulse when idle + zero balls? Alternatively we should also disable the first pulse. Just to keep it consistent.

Reply:
For all phases of ball search then?

Reply:
Took the second alternative and disabled the first pulse to keep it consistent.